src/libs/ease/array.c

```diff
@@ -48,44 +49,18 @@ int elektraArrayValidateName (const Key * key)
  */
 int elektraArrayValidateBaseNameString (const char * baseName)
```

Comment:
Sounds like this function is useless as it is basically identical to elektraIsArrayPart?